daemon/algod/api/server/v2/utils.go

```diff
@@ -206,18 +205,19 @@ func computeAppIndexFromTxn(tx node.TxnWithStatus, l LedgerForAPI) *uint64 {
 }
 
 // getCodecHandle converts a format string into the encoder + content type
-func getCodecHandle(formatPtr *string) (codec.Handle, string, error) {
-	format := "json"
+func getCodecHandle(formatPtr *generated.Format) (codec.Handle, string, error) {
+	format := generated.Json
 	if formatPtr != nil {
-		format = strings.ToLower(*formatPtr)
+		format = generated.PendingTransactionInformationParamsFormat(*formatPtr)
 	}
 
 	switch format {
-	case "json":
+	case generated.Json:
 		return protocol.JSONStrictHandle, "application/json", nil
-	case "msgpack":
+	case generated.Msgpack:
 		fallthrough
 	case "msgp":
+		// Can this actually ever happen?
```

Comment:
It looks like we should keep this for backwards compatibility - a client could call an API with the format set to `"msgp"` expecting a msgpack response. But it looks like this is never tested in the code in `handler_test.go`.